staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2983,6 +2983,13 @@ type PodSecurityContext struct {
 	// for that container.
 	// +optional
 	RunAsUser *int64 `json:"runAsUser,omitempty" protobuf:"varint,2,opt,name=runAsUser"`
+	// The GID to run the entrypoint of the container process.
```

Comment:
nit: prefer "The GID to run the container's init process"

(entrypoint is a docker term)

Reply:
this matches UID doc... I'd leave it for now